dev/AnimatedIcon/TestUI/AnimatedIconPage.xaml

```diff
@@ -539,6 +640,18 @@
                         </Viewbox>
                         <TextBlock x:Name="LastTransitionTextBlock"/>
                     </StackPanel>
+
+                    <StackPanel Background="Gray">
```

Comment:
nit: could move the style down into this StackPanel to keep the two together. its hard to know that the implicit button style about is what makes this scenario work.